internal/kafka/internal/workers/cluster_mgrs/deprovisioning_clusters_mgr.go

```diff
@@ -107,7 +107,7 @@ func (m *DeprovisioningClustersManager) reconcileDeprovisioningCluster(cluster *
 	}
 
 	// cluster has been removed from cluster service. Mark it for cleanup.
-	glog.Infof("Cluster %s  has been removed from cluster service.", cluster.ClusterID)
+	glog.Infof("Cluster %s has been removed from cluster service.", cluster.ClusterID)
```

Comment:
We've to add a logic in this reconciler to skip deprovisioning of enterprise clusters.

Additionally, we'll need a similar kind of logic for dynamic scale up/down reconcilers as well as we absolute shouldn't exercise any dynamic scaling logic for enterprise clusters. That could follow up in a separate JIRA.

Reply:
for dynamic scaling - the calculations exclude enterprise clusters (see https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1429/files#diff-c1b9a697a5a30e72d8cc46881d4360947c26e2b190bbb2cf65c8f52651cfe773R741)

Reply:
Ah apparently, I had missed that. Nice work, thanks!

Reply:
for the first bullet point - clusters returned in the function follows into `err = c.ClusterService.UpdateMultiClusterStatus(idsOfClustersToDeprovision, api.ClusterDeprovisioning)` are excluding cluster with `cluster_type` being `enterprise`